ls1intum/Artemis
# `Exam mode`: Add students section

### Checklist
#### General


#### Server
#### Client

### Motivation and Context



As part of my thesis, we add an additional students section to view the performed actions per student.

### Description


TODO


### Steps for Testing

Prerequisites:
- 1 Instructor
- One/Multiple Student
- 1 Exam

1. Log in to Artemis
2. Create an Exam with Exam Live Statistics enabled
3. Participate with the student
4. Verify that the student section shows the correct data

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


| Class/File | Branch | Line |
|------------|-------:|-----:|


TODO

### Screenshots



TODO
